pages/common/mosquitto_sub.md

```diff
@@ -0,0 +1,19 @@
+# mosquitto_sub
+
+> A simple MQTT version 3.1.1 client that will subscribe to topics and print the messages that it receives.
+
+- Subscribe to all broker status messages:
+
+`mosquitto_sub -v -t {{\$SYS/#}}`
+
+- Subscribe to "topic" sensors/temperature information on `localhost` with `QoS` 1:
+
+`mosquitto_sub -t {{sensors/temperature}} -q {{1}}`
+
+- Subscribe to temperature updates on multiple machines. This expects each machine to be publishing its temperature to "sensors/machines/`HOSTNAME`/temperature/`HD_NAME`":
```

Comment:
What's `HD_NAME`?

Reply:
`HD_NAME` relates to Hard Disk x; where `x` is the name of the harddisk